public/Button.ps1

```diff
@@ -54,7 +54,8 @@ function Button {
     )
 
     begin {
-        $buttonhash = Import-PowerShellDataFile $PSScriptRoot\internal\ButtonJsonTemplate.psd1
+        $scriptPath = split-path -parent $PSScriptRoot
+        $buttonhash = Import-PowerShellDataFile $scriptPath\private\ButtonJsonTemplate.psd1
```

Comment:
I think I must have renamed the folder from internal to private before publishing to the gallery so just changing from `internal` to `private` should be enough to fix the issue.

```suggestion
$buttonhash = Import-PowerShelldataFile $PSScriptRoot\Private\ButtonJsonTemplate.psd1
```

I will have to verify if this is the case though.